Tests/AblyChatTests/DefaultRoomPresenceTests.swift

```diff
@@ -0,0 +1,465 @@
+import Ably
+@testable import AblyChat
+import Testing
+
+struct DefaultRoomPresenceTests {
+    // MARK: CHA-PR1
+
+    // @spec CHA-PR1
+    @Test
+    func channelNameIsSetAsChatMessagesChannelName() async throws {
```

Comment:
This test doesn't seem to be testing anything. The class that's responsible for forming the channel name for a given feature is `DefaultRoom`, and I'd suggest testing CHA-PR1 the same way we test CHA-M1.

Reply:
removed - CHA-PR1 is not testable in specs